README.md

```diff
@@ -626,6 +628,7 @@ API | Description | Auth | HTTPS | CORS | Link |
 | Twitch | Game Streaming API | `OAuth` | Yes | Unknown | [Go!](https://dev.twitch.tv/docs) |
 | Twitter | Read and write Twitter data | `OAuth` | Yes | Unknown | [Go!](https://dev.twitter.com/rest/public) |
 | vk | Read and write vk data | `OAuth` | Yes | Unknown | [Go!](https://vk.com/dev/sites) |
+| Whatsapp | Send Whatsapp messages | No | Yes | Yes | [Go!](https://faq.whatsapp.com/en/android/26000030/) |
```

Comment:
This is not a stateless API; Rather, it is a unique URL generator for users of Whatsapp. Unfortunately this wont be appropriate for our collection here.